.github/workflows/test.yml

```diff
@@ -14,5 +14,5 @@ jobs:
         go-version: ${{ matrix.go-version }}
     - name: Checkout code
       uses: actions/checkout@v2
-    - name: Security
-      run: go get github.com/securego/gosec/cmd/gosec; `go env GOPATH`/bin/gosec ./...
+    - name: Test
+      run: go test -v
```

Comment:
I do like you using Actions in this project, but we shouldn't remove our verify security this project and it must work in all pull requests.

security.yml
```yml
on:
  pull_request:
    types: [opened, synchronize]

name: Security
jobs:
  test:
    strategy:
      matrix:
        go-version: [1.14.x]
        platform: [ubuntu-latest]
    runs-on: ${{ matrix.platform }}
    steps:
    - name: Install Go
      uses: actions/setup-go@v1
      with:
        go-version: ${{ matrix.go-version }}
    - name: Checkout code
      uses: actions/checkout@v2
    - name: Security
      run: go get github.com/securego/gosec/cmd/gosec; `go env GOPATH`/bin/gosec ./...

```